segtypes/n64/asm.py

```diff
@@ -27,7 +27,7 @@ def get_file_header():
 
     def scan(self, rom_bytes: bytes):
         if self.rom_start != "auto" and self.rom_end != "auto" and self.rom_start != self.rom_end:
-            self.funcs_text = self.disassemble_code(rom_bytes)
+            self.funcs_text: bool = self.disassemble_code(rom_bytes, options.get("asm_endlabels", False))
```

Comment:
funcs_text isn't a bool, it's a list of strings

Reply:
Whoops, fixed